packages/ethers5/src/client.ts

```diff
@@ -236,6 +236,7 @@ export class Web3Modal extends Web3ModalScaffold {
     this.options = options
 
     this.metadata = ethersConfig.metadata
+    this.setMetadata(this.metadata)
```

Comment:
You've now exposed metadata as option on createWeb3Modal, is this function still needed? I'm assuming we can now access metadata as and ordinary option and set it in scaffold.